packages/flutter/lib/src/material/constants.dart

```diff
@@ -4,14 +4,27 @@
 
 import 'package:flutter/painting.dart';
 
+/// The minimum dimension of any interactive region according to Material
+/// guidelines.
+///
+/// This is used to avoid small regions that are hard for the user to interact
+/// with. It applies to both dimensions of a region, so a square of size
+/// kMinInteractiveDimension x kMinInteractiveDimension is the smallest
+/// acceptable region that should respond to gestures.
+///
+/// See also:
+///
+///   * [kMinInteractiveDimensionCupertino]
+const double kMinInteractiveDimension = 48.0;
```

Comment:
This is covered in the "Touch and pointer size" section of the material spec: https://material.io/design/usability/accessibility.html

Probably should provide a link to it.